twisted/words/protocols/jabber/xmlstream.py

```diff
@@ -715,7 +727,7 @@ def upgradeWithIQResponseTracker(xs):
 
     This makes an L{XmlStream} object provide L{IIQResponseTracker}. When a
     response is an error iq stanza, the deferred has its errback invoked with a
-    failure that holds a L{StanzaException<error.StanzaException>} that is
+    failure that holds a L{error.StanzaError} that is
```

Comment:
The test is still supported ... for me error.StanzaError is not much better than StanzaError and it reduce the duplication :)

I have used shorted text :)